### ISSUE TITLE
Consider increasing default video-transcoding job concurrency

### ISSUE THREAD

**Issue by scanlime** [Chocobozzz/PeerTube#3659]
**Describe the problem to be solved**

Limiting transcodes to one at a time makes it very easy for long videos to result in unnecessarily long wait times for shorter videos, and it will under-utilize server CPUs that have many relatively slow cores.

I've been running diode.zone for most of its operation now with a very simple modification that sets the JOB_CONCURRENCY value for video-transcoding to a value larger than the normal setting of 1. I've found that 4 works well, but 2 would be a big improvement even. The main drawback to larger values here is increased RAM usage from the concurrent transcodes.

As of 3.0.0, I went back to using a vanilla build without this patch, in order to avoid contention with live streams, and it's been a very noticeable degradation in the overall transcode experience. Now that I see peertube is using distinct priority levels for live and vod transcodes, it might be a good time to allow an increase in the number of concurrent vod jobs.

(As an aside, if anyone asks why ffmpeg can't just use all the cores and do one job at a time.. beside the issue I mentioned where long jobs delay shorter jobs quite a lot, there's an issue with the maximum amount of concurrency that the codecs can actually reach in practice. If there are multiple jobs that can run in parallel, we get better hardware utilization by doing so.)

**Describe the solution you would like:**

At minimum, change the default job concurrency to 2, 3, or 4. Preferably, make the job concurrency easy for admins to change at runtime, or at least without recompiling from source.

**Comment by Chocobozzz** [comment]
Thanks for reporting this issue, but this is a duplicate of https://github.com/Chocobozzz/PeerTube/issues/951

**Comment by scanlime** [comment]
What do you think about increasing the default though? (the other issue was about adding an option)
